README.md

```diff
@@ -0,0 +1,88 @@
+# Chinese Newcomers Service Center
+This project is designed and developed by a team of UC Berkeley students through one of [Cal Blueprint](https://calblueprint.org/)'s project teams during the 2022-23 academic year. 
+
+Learn more about [Chinese Newcomers Service Center](https://chinesenewcomers.org/) and [Cal Blueprint](https://calblueprint.org/).
+
+---
+## Getting Started
+
+### Prerequisites
+
+Check your installation of `npm` and `node`:
+
+```sh
+node -v
+npm -v
+```
+
+This project requires node version 16.17.0.
+
+We strongly recommend using a Node version manager like [nvm](https://github.com/nvm-sh/nvm) (for Mac) or [nvm-windows](https://github.com/coreybutler/nvm-windows) (for Windows) to install Node.js and npm. See [Downloading and installing Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
+
+### Installation
+
+1. Clone the repo & install dependencies
+   1. Clone this repo
+      * using SSH (recommended)
+         ```sh
+         git clone git@github.com:calblueprint/chinese-newcomers.git
+         ```
+      * using HTTPS
+         ```sh
+         git clone https://github.com/calblueprint/chinese-newcomers.git
+         ```
+   2. Enter the cloned directory
+        ```sh
+        cd chinese-newcomers
+        ```
+   3. Install Typescript:
+        ```sh
+        npm install -g typescript
+        ```
+   4. Install yarn:
+        ```sh
+        npm install -g yarn
+        ```
+   5. Install project dependencies. This command installs all packages from [`yarn.lock`](yarn.lock).
+         ```sh
+         yarn install
+         ```
+2. Set up secrets:
+   1. In the project's root directory (`chinese-newcomers/`), create a new file named `.env`
+        ```sh
+        touch .env
+        ```
+   2. Copy the credentials from [Blueprint's internal Notion](https://www.notion.so/calblueprint/Firebase-Environment-Variables-c30a053fba4c47559f9f4944e4962a9f) (access is required) and paste them into the `.env` file.
```

Comment:
replace this v4v link w cnsc link: https://www.notion.so/calblueprint/Environment-Variables-2d24bb3a8b8c4b369fa620cbe1c8fe49?pvs=4

Reply:
fixed